cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -1137,3 +1138,25 @@ def test_mixed_security_group_overwrite_validator(head_node_security_groups, que
     )
     expected_message = "make sure.*cluster nodes are reachable" if expect_warning else None
     assert_failure_messages(actual_failures, expected_message)
+
+
+@pytest.mark.parametrize(
+    "root_volume_size, expected_message, ami_size",
```

Comment:
nit: I think it's more readable to have as first parameters our "variables" so root_volume_size and ami_size and as third parameter the value we're checking (expected_message).

Reply:
Done